UnityGLTF/Assets/UnityGLTF/Runtime/Plugins/GLTFSerialization/Schema/Skin.cs

```diff
@@ -30,6 +30,7 @@ public class Skin : GLTFChildOfRootProperty
 
 		public Skin()
 		{
+			Joints = new List<NodeId>();
```

Comment:
I wonder if this might have unintended consequences due to assumptions in this class and others.
This class itself specifically checks `Joints != null` on line 50, though it checks `Joints != null && Joints.Count > 0` on line 113. Maybe line 50 just needs to be updated to also check `.Count` (and potentially anywhere else in the codebase that accesses `Joints`)?